buildspec.saas.yml

```diff
@@ -44,6 +44,9 @@ phases:
       - yq eval -i ".rudder-transformer.image.tag=\"$VERSION\"" values.blue-release.yaml
       - yq eval -i ".rudder-transformer.image.tag=\"$VERSION\"" values.prod.yaml
       - git add values.blue-release.yaml values.prod.yaml
+      - cd ../../config-be-rudder-transformer
+      - yq eval -i ".image.tag=\"$VERSION\"" values.prod.yaml
+      - git add values.prod.yaml
```

Comment:
This will be auto synced?

Reply:
I believe. @psrikanth88, can you please confirm?

Reply:
@utsabc @saikumarrs yes, it has auto sync turned on